src/ClientHandle.cpp

```diff
@@ -1532,6 +1539,27 @@ void cClientHandle::HandleSlotSelected(Int16 a_SlotNum)
 
 
 
+void cClientHandle::HandleSpectate(const AString & a_PlayerUUID)
```

Comment:
No. But we need to think of a better way. The one I tried/am trying to implement is have a generic event object, which is dispatched to the relevant entities. Possibly with some form of observer/subscriber system.